packages/react/src/lib/useVoiceClient.ts

```diff
@@ -212,8 +212,11 @@ export const useVoiceClient = (props: {
 
   const sendSessionSettings = useCallback(
     (sessionSettings: Hume.empathicVoice.SessionSettings) => {
-      if (readyState !== VoiceReadyState.OPEN) {
-        throw new Error('Socket is not open');
+      if (
+        readyState !== VoiceReadyState.OPEN &&
+        readyState !== VoiceReadyState.IDLE
```

Comment:
why would we allow sending session settings if the websocket is idle?

Reply:
The first session settings from `VoiceProvider` prop is sent in the `connect` callback, which is run before `readyState` has had time to update. I've now changed the logic so that instead of using the connect callback, it will be sent on `readyState` being `OPEN` for the first time.